src/test/java/com/google/sps/CommentAnalysisTest.java

```diff
@@ -0,0 +1,85 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps;
+
+import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport;
+import com.google.api.client.http.javanet.NetHttpTransport;
+import com.google.api.client.json.JsonFactory;
+import com.google.api.client.json.jackson2.JacksonFactory;
+import com.google.api.services.youtube.YouTube;
+import com.google.api.services.youtube.model.CommentThreadListResponse;
+import com.google.sps.servlets.utils.CommentAnalysis;
+import com.google.sps.servlets.utils.Statistics;
+import java.io.IOException;
+import java.security.GeneralSecurityException;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/**
+ * This is a JUnit test for sentiment analysis
+ */
+@RunWith(JUnit4.class)
+public class CommentAnalysisTest {
```

Comment:
Thanks for adding the unit test!
I see that the test actually calls the youtube API. Do you think you could mock out the service by using Mokito(or other testing framework) so we can keep the test as independent as possible?

Reply:
Will try that right now!

Reply:
Talked offline, this change will be implemented in the merge branch.